Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -972,16 +973,20 @@ void ABenchmarkGymGameModeBase::UpdateAndCheckTotalActorCounts()
 		{
 			// Check for test failure
 			const FExpectedActorCountConfig& ExpectedActorCount = ExpectedActorCounts[ActorClass];
-			bActorCountFailureState = TotalActorCount < ExpectedActorCount.MinCount || TotalActorCount > ExpectedActorCount.MaxCount;
-			if (bActorCountFailureState && !bHasActorCountFailed)
+			bool bActorCountFailed = TotalActorCount < ExpectedActorCount.MinCount || TotalActorCount > ExpectedActorCount.MaxCount;
```

Comment:
As an alternative to reduce the number of similarly named variabled... we might be able to reset bActorCountFailureState to false as you do but do:

bActorCountFailureState |=TotalActorCount < ExpectedActorCount.MinCount || TotalActorCount > ExpectedActorCount.MaxCount;

Think this should work and keep the rest of the code the same?

Reply:
Reworked from @simonsarginson comments already - pt another look